File: src/pt/joaomneto/titancompanion/adventurecreation/impl/COHAdventureCreation.java
package pt.joaomneto.titancompanion.adventurecreation.impl;

import java.io.BufferedWriter;
import java.io.IOException;

import pt.joaomneto.titancompanion.R;
import pt.joaomneto.titancompanion.adventure.Adventure;

/**
 * Created by Cristina on 28-07-2015.
 */
public class COHAdventureCreation  extends TFODAdventureCreation{

    public COHAdventureCreation() {
        super();
        fragmentConfiguration.clear();
        fragmentConfiguration.put(0, new Adventure.AdventureFragmentRunner(R.string.title_adventure_creation_vitalstats,
                "pt.joaomneto.titancompanion.adventurecreation.impl.fragments.VitalStatisticsFragment"));

    }

    @Override
    protected void storeAdventureSpecificValuesInFile(BufferedWriter bw)
            throws IOException {
        bw.write("gold=0\n");
    }
}

File: src/main/java/pt/joaomneto/titancompanion/adventurecreation/impl/COHAdventureCreation.kt
package pt.joaomneto.titancompanion.adventurecreation.impl

import android.view.View
import java.io.BufferedWriter
import java.io.IOException
import pt.joaomneto.titancompanion.adventurecreation.AdventureCreation

/**
 * Created by Cristina on 28-07-2015.
 */
class COHAdventureCreation : AdventureCreation() {

    override fun rollGamebookSpecificStats(view: View) {
    }

    override fun validateCreationSpecificParameters(): String? {
        return null
    }

    @Throws(IOException::class)
    override fun storeAdventureSpecificValuesInFile(bw: BufferedWriter) {
        bw.write("gold=0\n")
    }
}
